tuxemon/event/actions/teleport_faint.py

```diff
@@ -51,10 +53,12 @@ class TeleportFaintAction(EventAction[TeleportFaintActionParameters]):
     def start(self) -> None:
         player = self.session.player
 
-        # Start with the default value, override if game variable exists
-        teleport = ["healing_center.tmx", 7, 10]
+        # If game variable exists, then teleport:
         if "teleport_faint" in player.game_variables:
             teleport = player.game_variables["teleport_faint"].split(" ")
+        else:
+            logger.error(f"Old savegame. Please heal your monsters in a center to fix it.")
+            return None
```

Comment:
Bit picky, but as the calling function isn't expecting to receive a value from this method, I would use `return` rather than `return None`

Reply:
Fair point, I'll fix it